integration-tests/contract-no-annotations-call-by-index.spec.ts

```diff
@@ -3,13 +3,13 @@ import { noAnnotCode, noAnnotInit } from "./data/token_without_annotation";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract with no annotations calling methods by index using: ${rpc}`, () => {
+  describe(`Test contract origination with no annotations calling methods by index through contract api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('Test contract with no annotations for methods', async (done) => {
+    it('Verify contract.originate for a contract with no annotations for method', async (done) => {
```

Comment:
It is not the origination itself that we want to test, but if we are able to call an entrypoint using `method` when the contract has no annotation (so using index).